addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-config-vars-service.js

```diff
@@ -211,6 +227,18 @@ class EnvironmentConfigVarsService extends Service {
       memberAccountId: accountId,
     });
 
+    let egressStoreIamPolicyDocument;
+    const enableEgressStore = this.settings.get(settingKeys.enableEgressStore);
+    if (enableEgressStore) {
+      const egressStoreMount = await this.getEgressStoreMount(requestContext, environment);
+      s3Mounts.push(egressStoreMount);
+      egressStoreIamPolicyDocument = await this.getEnvEgressStorePolicy(requestContext, {
+        environment,
+        egressStore: egressStoreMount,
+        memberAccountId: accountId,
+      });
+    }
+
```

Comment:
Add test for this branch?

Reply:
Tests added in commit: c0dfd0b9e97060702c0940a873a3f18d29941584